Add hourly solar curtailment chart

noracato opened the pull request:
I believe the solar_curtailment chart is hereby finished. Thanks again @antw for helping with the chart bugs

marliekeverweij commented:
I love it!

I found one issue. However, I cannot reproduce it, so don't know what to do about it
![Screen Shot 2020-03-18 at 09 15 42](https://user-images.githubusercontent.com/32833996/76942757-8d4f7c80-68fe-11ea-8d10-88a194b383af.png)

noracato commented:
@antw Thanks! I incorporated your suggested changes and removed as many 'this' as possible. I agree it's much easier to read the code now.
@marliekeverweij It looks like the housholds are plotted for a week view instead of a year view. Could this have happened after switching options too fast?

marliekeverweij commented:
> It looks like the housholds are plotted for a week view instead of a year view. Could this have happened after switching options too fast?

Yes that could be it!
